charts/hivemq-platform-operator/values.yaml

```diff
@@ -42,6 +42,18 @@ serviceAccount:
   # The service account name to be used, or "hivemq-<release name>" if not set.
   name: ""
 
+# Configures the service account and permissions the operator creates for the HiveMQ Platform Pods.
+platformServiceAccount:
+  # Specifies whether a service account should be created for each HiveMQ Platform.
+  create: true
+  # Specifies whether the service account for the HiveMQ Platform should be validated.
+  validate: true
```

Comment:
Is it necessary to make "validate" configurable? What if we always validate no matter what?

Reply:
Then we always need the clusterroles and clusterrolebindings get/list/watch permissions. I want to make them configurable, so customers can limit the permissions the operator needs.

For now I cannot limit the read permissions further, since we still have a dependent resource for ServiceAccount, Role and RoleBinding. But this might change in the future. Then the read permissions for those resource could also be made configurable/optional (when validation is not configured).